### PR TITLE
Add git contributros page to the application

### PR DESCRIPTION
**Pull request by shubhagarwal1**
#198 
Screenshot 2024-10-26 at 9 48 56 AM

### CODE REVIEW

**Review by ajaynegi45 — CHANGES_REQUESTED** [review]
Hi @ ,

Thank you for your prompt work on this issue. I appreciate your dedication to the project.

I have reviewed your work, and I need some changes.

- Improve UI more
- Add more Things

Once again, thank you for your contribution! ❤️

File: src/app/contributors/page.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { Loader2 } from "lucide-react";
```
[the remark this answers is not in the document]
Comment (shubhagarwal1) [reply src/app/contributors/page.tsx:4]: hey can i work on thi s?

[the remark this answers is not in the document]
Comment (ajaynegi45) [reply src/app/contributors/page.tsx:4]: > hey can i work on thi s?

Yes

[the remark this answers is not in the document]
Comment (shubhagarwal1) [reply src/app/contributors/page.tsx:4]: please suggest meif any changes are required

**Pull request closed without merging**